### PR TITLE
Add long explanation for E0547

### PR DESCRIPTION
**Pull request by jesusprubio**
Helps with #61137

### CODE REVIEW

File: compiler/rustc_error_codes/src/error_codes/E0547.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#![stable(since = "1.0.0", feature = "test")]
```
Comment (GuillaumeGomez) [inline compiler/rustc_error_codes/src/error_codes/E0547.md:1]: I feel like a big part of the file "disappeared". :rofl:

File: compiler/rustc_error_codes/src/error_codes/E0547.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+#[unstable(feature = "_unstable_fn")] // invalid
+fn _unstable_fn() {}
+
+#[rustc_const_unstable(feature = "_unstable_const_fn")] // invalid
+fn _unstable_const_fn() {}
+```
+
+To fix the issue you need to provide the `issue` field.
```
Comment (GuillaumeGomez) [inline compiler/rustc_error_codes/src/error_codes/E0547.md:16]:
```suggestion
To fix this issue, you need to provide the `issue` field. Example:
```

File: compiler/rustc_error_codes/src/error_codes/E0547.md (hunk 1 above)
Comment (jesusprubio) [reply compiler/rustc_error_codes/src/error_codes/E0547.md:7]: This was strange xD

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup squash

**Pull request merged**